spring-kafka/src/test/java/org/springframework/kafka/retrytopic/AsyncMonoRetryTopicScenarioTests.java

```diff
@@ -0,0 +1,1241 @@
+/*
+ * Copyright 2018-2024 the original author or authors.
```

Comment:
I don't think that such a past is correct to indicate when this class was created: https://stackoverflow.com/questions/9039698/which-years-to-write-into-copyright-header.

Someone used the first year as the one when project has started, but that still is not `2018`.
Therefore I suggest to do just `2024`.
When we change this class in `2025`, some our tool is going to do for us in this class `2024-2025`.
This way we would follow jurisdiction: the year when entity was created and the year when entity was last changed.

Please, see if all other your new classes have to be fixed respectively.